tests/system/Models/UpdateModelTest.php

```diff
@@ -260,6 +260,26 @@ public function testUpdateObjectWithDataException(): void
 		$this->createModel(EventModel::class);
 
 		$data = (object) [
+							 'name'    => 'Foo',
+							 'email'   => 'foo@example.com',
+							 'country' => 'US',
+							 'deleted' => 0,
+						 ];
```

Comment:
@paulbalandan Is this indentation intended? It is appplied by the pre-commit hook with cs-fixer.

Reply:
codesniffer is stubbornly wrong on fixing array indentations, especially these types when the array has something before the opening bracket.

I suggest fixing all other things codesniffer fixes, but do not follow on array indentations using `--no-verify` on commit

Reply:
Alright thanks. I will need to go back to command line committing then, got used to Github Desktop quite too much 😆